app/src/main/java/com/simplecity/amp_library/ui/views/multisheet/MultiSheetSlideEventRelay.java

```diff
@@ -1,19 +1,14 @@
 package com.simplecity.amp_library.ui.views.multisheet;
 
 import android.support.design.widget.BottomSheetBehavior;
-
 import com.jakewharton.rxrelay2.BehaviorRelay;
 import com.simplecity.multisheetview.ui.view.MultiSheetView.Sheet;
-
-import javax.inject.Inject;
-
 import io.reactivex.Observable;
 
 public class MultiSheetSlideEventRelay {
 
     private final BehaviorRelay<SlideEvent> eventRelay = BehaviorRelay.create();
 
-    @Inject
```

Comment:
What's the purpose of this change?

Reply:
I wanted to start flattening the DI structured, but you are right, better to do it separately. I just put the Injection back but removed the unnecessary Module.